Automattic/wp-calypso
# Fix/hide atomic credentials

#### Changes proposed in this Pull Request

* Check for atomic credentials and disable editing and revoking

#### Testing instructions



* Ensure that business plans credentials are connected and cannot be edited or revoked
* Ensure that stand alone sites can be edited and revoked using the step by step credentials



Related to #62580 & #60820
